### PR TITLE
OCPBUGS-33080: Update EQoS CRD to latest

### PR DESCRIPTION
**Pull request by tssurya**
Bring in the changes from https://github.com/ovn-org/ovn-kubernetes/pull/4158 for 4.16

cc @pperiyasamy

### CODE REVIEW

File: bindata/network/ovn-kubernetes/common/001-crd.yaml [hunk 1]
```diff
@@ -316,12 +316,12 @@ status:
   conditions: []
   storedVersions: []
 ---
+---
```
Comment (pperiyasamy) [inline bindata/network/ovn-kubernetes/common/001-crd.yaml:319]: nit: can you remove this extra separator ?

Comment (tssurya) [reply bindata/network/ovn-kubernetes/common/001-crd.yaml:319]: oh woops haha

Comment (tssurya) [reply bindata/network/ovn-kubernetes/common/001-crd.yaml]: done! thanks for catching that Peri!

**Comment by kyrtapz** [comment]
@pperiyasamy we could use some downstream e2e tests in origin for this feature to avoid bugs like this one.
/lgtm

**Pull request merged**